scalafmt-core/shared/src/main/scala/org/scalafmt/config/RedundantBracesSettings.scala

```diff
@@ -8,7 +8,8 @@ case class RedundantBracesSettings(
     maxLines: Int = 100,
     stringInterpolation: Boolean = false,
     parensForOneLineApply: Option[Boolean] = None,
-    generalExpressions: Boolean = true
+    generalExpressions: Boolean = true,
+    ifElseExpressions: Boolean = false
```

Comment:
Not sure about default bu removing braces around if statements look rare desire

Reply:
i think the default is fine.

however, i think one of the requests was to be able to rewrite `if` statements consistently; that is, either rewrite both `if` and `else`, or keep both unchanged (that might mean the flag is not just boolean). i don't know how difficult it would be to implement this logic.

also, in #2038, they want to rewrite assignments but not unit expressions... not suggesting to add it here but simply asking whether it will be simple to do that with the current approach.

Reply:
I can't estimate the complexity of consistent and assignment formatting too. At first glance most users will be satisfied with this simple flag